backend/tests/test_api.py

```diff
@@ -1,4 +1,42 @@
+import pytest
+from backend.editor.api import api
+from falcon import testing
+
+
+@pytest.fixture(autouse=True)
+def test_setup(neo4j):
+    # delete all the nodes and relations in the database
+    query = "MATCH n DETACH DELETE n"
+    neo4j.session().run(query)
+    query = "DROP INDEX p_test_branch_SearchIds IF EXISTS"
+    neo4j.session().run(query)
+    query = "DROP INDEX p_test_branch_SearchTags IF EXISTS"
+    neo4j.session().run(query)
+
+@pytest.fixture()
+def client():
+    return testing.TestClient(api)
+
 def test_hello(client):
     response = client.get("/")
     assert response.status_code == 200
     assert response.json() == {"message": "Hello user! Tip: open /docs or /redoc for documentation"}
+
+def delete_project_test(neo4j):
+    session = neo4j.session()
+    query = "MATCH (n:p_test_branch:t_test:b_branch:ENTRY) RETURN COUNT(*)"
```

Comment:
This query isn't being used anywhere. Please remove it.

Reply:
Done